src/content/docs/kubernetes-pixie/kubernetes-integration/get-started/introduction-kubernetes-integration.mdx

```diff
@@ -20,56 +20,51 @@ redirects:
 
-## Why it matters [#features]
+Take a look at what you can do with New Relic:
+
+* You can monitor the [Kubernetes control plane](/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/configure-control-plane-monitoring) to manage and collect metrics from your clusters.
 
-Governing the complexity of Kubernetes can be challenging; there's so much going on at any given moment, with containers being created and deleted in a matter of minutes, applications crashing, and resources being consumed unexpectedly. Our integration helps you navigate Kubernetes abstractions across on-premises, cloud, and hybrid deployments.
+* You can [forward your Kubernetes logs](/docs/logs/forward-logs/kubernetes-plugin-log-forwarding/) to New Relic, which will give you more ways to manage your log data. You can collect, process, explore, query, and alert on your log data.
 
-In New Relic, you can build your own charts and [query](/docs/using-new-relic/data/understand-data/query-new-relic-data) all your [Kubernetes data](/docs/integrations/kubernetes-integration/understand-use-data/find-use-your-kubernetes-data), which our integration collects by instrumenting the container orchestration layer. This gives you additional insight into nodes, namespaces, deployments, replica sets, pods, and containers.
+* You can build your own charts and [query](/docs/using-new-relic/data/understand-data/query-new-relic-data) all your [Kubernetes data](/docs/integrations/kubernetes-integration/understand-use-data/find-use-your-kubernetes-data) that our integration collects by instrumenting the container orchestration layer. This gives you more insight into nodes, namespaces, deployments, replica sets, pods, and containers.
 
-<img
-  title="New Relic - Kubernetes dashboard"
-  alt="New Relic - Kubernetes dashboard"
-  src={kubernetesDashboard}
-/>
+* You can [link your APM data to Kubernetes](/docs/integrations/kubernetes-integration/link-your-applications/link-your-applications-kubernetes) to measure how well your web and mobile apps are performing. You can track things like request rate, throughput, error rate, and availability.
 
-<figcaption>
-  <DoNotTranslate>**[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Dashboards**</DoNotTranslate>: Using the [query builder](/docs/chart-builder/use-chart-builder/get-started/introduction-chart-builder) you can turn any query on Kubernetes data to clear visuals.
-</figcaption>
+* You can [monitor services running on Kubernetes](/docs/integrations/kubernetes-integration/link-apps-services/monitor-services-running-kubernetes), like Apache, NGINX, Cassandra, and many more. Check out our [tutorial for monitoring Redis on Kubernetes](/docs/integrations/kubernetes-integration/link-apps-services/tutorial-monitor-redis-running-kubernetes).
```

Comment:
I would add another point after this one

* You can easily scrape Prometheus metrics from any workload in the cluster thanks to the service discovery provided by the [Prometheus Agent](https://docs.newrelic.com/docs/infrastructure/prometheus-integrations/get-started/send-prometheus-metric-data-new-relic/#Agent).